### ISSUE TITLE
Remove CWD from filePath

### ISSUE THREAD

**Issue by furzeface** [furzeface/grunt-xml-sitemap#7]

**Comment by arminrosu** [comment]
This worked for me, added it to tasks/xml_sitemap.js#L56. Sorry, to little a change to warrant a fork:
```js
// remove CWD from filename
if (file.indexOf(options.dest) === 0) {
	file = file.slice(options.dest.length);
}
```

**Comment by MrHyde** [comment]
This one is the bigest issue atm, but it requires changes in tests aswell. I'll try to deliver new version untill the end of the week.

**Comment by MrHyde** [comment]
Since file destination and cwd might be different, solution mentioned above works only in very few cases. Better fix was offered in #13
